data/102/064/053/102064053.geojson

```diff
@@ -18,12 +24,6 @@
     ],
     "lbl:latitude":53.569433,
     "lbl:longitude":10.04852,
-    "misc:deu_x_preferred_longname":[
-        "Freie und Hansestadt Hamburg"
-    ],
-    "misc:deu_x_preferred_placetype":[
-        "freie und hansestadt"
-    ],
     "mps:latitude":53.569433,
     "mps:longitude":10.04852,
     "mz:hierarchy_label":1,
```

Comment:
General question for big unitary cities in Germany, should the `county` placetype record for that place have the `hiearachy_label` = 0?

Reply:
Yeah, ideally we'd set the counties with hierarchy_label 0 to be consistent. I'll adjust in a new commit.

Reply:
There are 100 cases of coterminous locality/county records in Germany.

An upcoming commit will set the hierarchy_label to 0 for the county records and flag the pairs as coterminous.